### ISSUE TITLE
regression: glow gets confused and outputs ansi gibberish under uxrvt

### ISSUE THREAD

**Issue by dch** [charmbracelet/glow#276]
I guess this isn't a glow issue but you never know...

glow works perfectly under xterm, alacritty & wezterm, and when run *without* a direct path parameter.

- tested against head 1f2b3671c715f4a84d8103f617ee35d821fb347b & `1.4.1`
- `x11/rxvt-unicode` on `FreeBSD 13.0-RELEASE amd64`
- TERM=xterm-256color

### with pager set in glow config

```
dch@wintermute /p/c/serenity> glow README.md 
^[]11;rgba:0000/0000/01ff/c800^[^[[3;1R
   Serenity  <------------- back in "nice" glow mode again
...
                
...skipping...

228;48;5;63;1mSerenity <------ oh dear the charm is leaking everywhere
;5;252m                                                                       
r:                                               
2m... gets worse and worse
```

### without pager

```
dch@wintermute /p/c/serenity> glow README.md
^[]11;rgba:0000/0000/01ff/c800^[^[[3;1R
   Serenity  <--------- back in "nice" glow mode again                              
                                                                              
...
            ## deploy via ansible                                                       
                                                                              
  Read the  Makefile  to see exactly what roles/steps are taken behind the    
  scenes:                                                                     
                                                                            
    $ make facts sign serenity                                                

<---------- perfect until the final lifeless gasp, whereupon coloured sputum attempts
to bypass the terminal...
dch@wintermute /p/c/serenity> 00[3;1R]11;rgba:0000/0000/01ff/c800[3;1R
```

**Comment by dch** [comment]
Seems this commit broke this:

```
$ git bisect good
5b29a023bf541323ee64d5cbe0d967a689a9e611 is the first bad commit
commit 5b29a023bf541323ee64d5cbe0d967a689a9e611
Author: Christian Muehlhaeuser <muesli@gmail.com>
Date:   Thu Mar 11 16:33:53 2021 +0100

    Bump charm, babyenv, termenv, cobra, and viper dependencies

 go.mod | 10 +++++-----
 go.sum | 37 +++++++++++++++----------------------
 2 files changed, 20 insertions(+), 27 deletions(-)
```

**Comment by muesli** [comment]
Which version of urxvt are you running? I can't reproduce this with 9.26.

**Comment by dch** [comment]
9.22_4

**Comment by dch** [comment]
looks like FreeBSD wants a few patches to bump rxvt-unicode, I'll update you tomorrow on this. What distro are you running urxvt on?

**Comment by dch** [comment]
@muesli thanks, indeed bumping urxvt resolves this

**Comment by muesli** [comment]
@dch Thanks for confirming this! :heart:

**Comment by RuiApostolo** [comment]
Version 9.22 is the latest rxvt-unicode available on Ubuntu, including 21.10. Any chance this could be revisited?
